popochiu/Characters/Goddiu/CharacterGoddiu.gd

```diff
@@ -8,6 +8,9 @@ const Data := preload('CharacterGoddiuState.gd')
 
 var state: Data = preload('CharacterGoddiu.tres')
 
+# turns anti-glide animation on.
+# will not work if call method track in animation player is not calling update_position every frame
+export var anti_glide_animation: bool = false
```

Comment:
Not needed if you move the variable to `PopochiuCharacter.gd` as mentioned in the comment for the `CharacterTemplate.gd` file.

This comment also applies for `CharacterPopsy.gd` and `CharacterTrapusinsiu.gd`.